lang/ilias_en.lang

```diff
@@ -17089,11 +17089,11 @@ trac#:#user_total#:#User Total
 trac#:#view_mode#:#View Mode
 tstv#:#tstv_create#:#Create Test Certificate
 tstv#:#tstv_create_info#:#Select a completed test to generate a certificate for it.
-ui#:#datatable_multiactionmodal_actionlabel#:#operation
+ui#:#datatable_multiactionmodal_actionlabel#:#Action for all entries
 ui#:#datatable_multiactionmodal_buttonlabel#:#Execute
-ui#:#datatable_multiactionmodal_listentry#:#apply to all objects
-ui#:#datatable_multiactionmodal_msg#:#<b>Caution!</b> multiple objects affected.
-ui#:#datatable_multiactionmodal_title#:#Operation on multiple objects
+ui#:#datatable_multiactionmodal_listentry#:#Actions for Complete Table
+ui#:#datatable_multiactionmodal_msg#:#An action selected will affect all entries in this table.
```

Comment:
Better: 'Selected action will affect all entries in this table.'

Reply:
Is adjusted